docs/versioned_docs/version-2.11.0/stateful-applications/cassandra.md

```diff
@@ -0,0 +1,234 @@
+
+  ```
+  $kubectl get pod,service,sts,pvc -n cassandra
+  
+  NAME                                    READY   STATUS    RESTARTS   AGE
+  cassandra-openebs-node-0   2/2          Running     0                4m
+  cassandra-openebs-node-1   2/2          Running     0                3m2s
+  cassandra-openebs-node-2   2/2          Running     0                3m24s
+
+  NAME                         READY   AGE
+  statefulset.apps/cassandra   3/3     6m35s
+
+  NAME                TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)                                        AGE
+  service/cassandra-openebs-svc   ClusterIP   None         <none>        7000/TCP,7001/TCP,7199/TCP,9042/TCP,9160/TCP   6m35s
+  
+  NAME                                         STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS     AGE
+  var-lib-cassandra-cassandra-openebs-node-0   Bound    pvc-213f2cfb-231f-4f14-be93-69c3d1c6d5d7   20Gi       RWO            openebs-device   20m
+  var-lib-cassandra-cassandra-openebs-node-1   Bound    pvc-059bf24b-3546-43f3-aa01-3a6bea640ffd   20Gi       RWO            openebs-device   19m
+  var-lib-cassandra-cassandra-openebs-node-2   Bound    pvc-82367756-7a19-4f7f-9e35-65e7696f3b86   20Gi       RWO            openebs-device   18m
+  ```
+
+- Login to one of the Cassandra pod to verify the Cassandra cluster health status using the following command.
+
+  ```
+  $ kubectl exec -it cassandra-openebs-node-0 bash -n cassandra
+  
+  cassandra@cassandra-openebs-node-0:/$ nodetool status
+  Datacenter: datacenter1
+  =======================
+  Status=Up/Down   |/ State=Normal/Leaving/Joining/Moving      --  Address        Load       Tokens       Owns (effective)  Host ID                               Rack
+  UN  192.168.30.24  94.21 KiB  256          63.0%             73c54856-f045-48db-b0db-e6a751d005f8  rack1
+  UN  192.168.93.31  75.12 KiB  256          65.3%             d48c61b7-551b-4805-b8cc-b915d039f298  rack1
+  UN  192.168.56.80  75 KiB     256          71.7%             91fc4107-e447-4605-8cbf-3916f9fd8abf  rack1
+  ```
+
+- Create a Test Keyspace with Tables. Login to one of the Cassandra pod and run the following commands from a cassandra pod.
+
+  ```
+  cassandra@cassandra-openebs-node-0:/$ cqlsh <svc-name>.<namespace>.svc.cluster.local
+  ```
+
+  Example command:
+
+  ```
+  cassandra@cassandra-openebs-node-0:/$ cqlsh cassandra-openebs-svc.cassandra.svc.cluster.local
+  
+  Connected to cassandra-openebs at cassandra-openebs-svc.cassandra.svc.cluster.local:9042.
+  [cqlsh 5.0.1 | Cassandra 3.11.6 | CQL spec 3.4.4 | Native protocol v4]
+  Use HELP for help.
+  cqlsh>
+  ```
+
+- Creating a Keyspace. Now, let’s create a Keyspace and add a table with some entries into it.
+
+  ```
+  cqlsh> create keyspace dev
+  ... with replication = {'class':'SimpleStrategy','replication_factor':1};
+  
```

Comment:
please insert ` ``` `

Reply:
we dont have to close `` ``` `` unless there is sibling or child content to the code block, in this case the parent content end after code block

Reply:
why are we using this mixed type of rule. somewhere including ` ``` ` and somewhere not including ` ``` `

Reply:
It is better to be consistent.

Reply:
@IsAmrish It is taken from existing docs.